src/components/WebexMeetingIf/WebexMeetingIf.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {useMeeting} from '../hooks';
+
+/**
+ * WebexMeetingIf
+ *
```

Comment:
This helper component makes it easy for developers to add content conditionally, depending on the current meeting's state, without obtaining the meeting object with the adapter `getMeeting()` function or the `useMeeting()` hook.

Example:
```js
<WebexMeetingIf meetingID="someid" condition={meeting => meeting.state !== 'LEFT'}>
  <WebexMeetingControlBar controls={...} />
</WebexMeetingIf>
```